ivy_tests/test_ivy/test_functional/test_nn/test_activations.py

```diff
@@ -116,16 +116,17 @@ def test_sigmoid(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
 @handle_test(
     fn_tree="functional.ivy.softmax",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
         min_num_dims=1,
         large_abs_safety_factor=8,
-        small_abs_safety_factor=8,
+        small_abs_safety_factor=4,
```

Comment:
I see you reduced the safety factor, was there a reason for this?

Reply:
I was trying to solve the errors that appeared and I tried this safety factor and it worked for me